src/js/_store/modules/categories.js

```diff
@@ -5,14 +5,33 @@ export default {
   state: {
     categories: [],
     errorMessage: '',
+    newCategoryName: '',
+    creatingCategory: false,
```

Comment:
[imo]

state で管理するようにして複数のコンポーネントから参照すれば良いだけにしてくれたと思うので、汎用的な命名にした方が良いかなと思います！
`isLoading` とかが一般的に使用されるかと思います！

```suggestion
    isLoading: false,
```

Reply:
汎用的な命名にするように癖づけします。変更しました！
c94737a617f5aa376eede8b8d645e2485efc29bf